pkg/workload/admissionchecks.go

```diff
@@ -86,6 +86,19 @@ func FindAdmissionCheck(checks []kueue.AdmissionCheckState, checkName string) *k
 	return nil
 }
 
+// SetAllChecksToPending sets all AdmissionChecks to Pending
+func SetAllChecksToPending(w *kueue.Workload, message string) {
+	checks := w.Status.AdmissionChecks
+	for i := range checks {
+		checks[i] = kueue.AdmissionCheckState{
+			Name:               checks[i].Name,
+			State:              kueue.CheckStatePending,
+			LastTransitionTime: metav1.NewTime(time.Now()),
+			Message:            message,
```

Comment:
Maybe just say "Reset to Pending due to eviction. Previously: <reason>"

Reply:
Done